### PR TITLE
fix bug (cutoff), add unit tests, lock version for yolk

### PR DESCRIPTION
**Pull request by gilzellner**

### CODE REVIEW

File: python_dependency_verifier/tests/unittests.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+class TestDependencies(unittest.TestCase):
+
+    def runTest(self):
+        pass
+
+    def check_file(self):
+        return self.assertEqual(
+            python_dependency_verifier.check_dependencies_file(
+                "../../setup.py", "cloudify.*"),
```
Comment (nir0s) [inline python_dependency_verifier/tests/unittests.py]: You're using the main setup.py file. What if it changed and you add another dependency?
You should probably create a resources directory and put a mock setup.py file there.

**Comment by nir0s** [comment]
You should create a .travis file and add the tests there. If you want to support multiple Python versions (which you probably should as this is a very generic tool), add tox as well. You currently set the tests to run manually.

**Comment by nir0s** [comment]
Additionally, I don't know what you meant to do when you started writing this.. but you've pretty much written "integration tests' as you're not really testing units here.

File: python_dependency_verifier/tests/unittests.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import unittest
+from python_dependency_verifier import python_dependency_verifier
+
+
+class TestDependencies(unittest.TestCase):
+
+    def runTest(self):
+        pass
+
+    def check_file(self):
```
Comment (nir0s) [inline python_dependency_verifier/tests/unittests.py]: for nose to be able to run the tests they have to be named "test_*"

Comment (gilzellner) [inline python_dependency_verifier/tests/unittests.py]: done

File: python_dependency_verifier/tests/unittests.py (hunk 1 above)
Comment (gilzellner) [inline python_dependency_verifier/tests/unittests.py]: done

**Comment by gilzellner** [comment]
added tox, travis

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# content of: tox.ini , put in same dir as setup.py
+[tox]
+envlist=flake8,py27,py26
+
+[testenv:py26]
+deps =
+    -rdev-requirements.txt
+commands=nosetests --with-cov --cov-report term-missing --cov repex python_dependency_verifier/tests -v
```
Comment (nir0s) [inline tox.ini]: repex?

**Pull request merged**